docs/pages/experiments/md3/index.tsx

```diff
@@ -53,20 +53,27 @@ export const customPalette = {
   },
   neutral: {
     '0': '#000000',
+    '4': '#0f0d14',
+    '6': '#15121a',
     '10': '#201a1c',
+    '12': '#211e26',
     '17': '#2e282a',
     '20': '#352f30',
     '22': '#393335',
+    '24': '#3b3840',
     '30': '#4c4546',
     '40': '#645c5e',
     '50': '#7e7577',
     '60': '#988e90',
     '70': '#b3a9aa',
     '80': '#cfc4c5',
+    '87': '#ded8e3',
     '90': '#ebe0e1',
     '92': '#f1e5e6',
+    '94': '#f2ebf7',
     '95': '#faeef0',
     '96': '#f7f2fa',
+    '98': '#fef7ff',
     '99': '#fffbff',
     '100': '#ffffff',
```

Comment:
As you mentioned, only a few tones are used in the 100-tone spectrum. I also tried to use the `@material\material-color-utilities` to generate everything at once on my project, but it didn't generate all needed tones. The only solution that worked to me is the [Figma Theme Builder](https://www.figma.com/community/plugin/1034969338659738588/material-theme-builder) tool. 

An interesting question to make is if it worth to create an automated way to generate the colors. My first impression is that, it is a better idea to keep it static, decreasing the processing needed and the chances for unexpected errors.

A possible alternative is to let the type more generic:
```
type neutral = {[key: number]: string}
```

And, maybe, accepting the function `create[Light|Dark|Custom]ColorScheme`, that is responsible to connect the tones with the color scheme, as parameter on the `extendTheme().colorSchemes` configs.

But it is definitely an important problem to understand how color schema mapping can be generalized. However, expanding the component library sounds a bigger priority for now, since it blocks people to easily use material-next.